### PR TITLE
Product Concepts doc

### PR DESCRIPTION
**Pull request by andyjih**

### CODE REVIEW

File: documents/product/concepts.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# Grouparoo Product Concepts
+
+## Profiles
+
+Profiles are the primary data object in Grouparoo. Profiles represent individual people such as:
```
Comment (evantahler) [inline documents/product/concepts.md]: > Profiles represent individual people.  For your specific company, that may mean:
...

File: documents/product/concepts.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+- Customers
+- Users
+- Visitors to your site or app
+
+## Profile Properties
+
+Profile Properties are data associated with a profile. You can define Profile Properties based on data that exists in your data Sources.
+
+There are two types of Profile Properties: `Unique` and `Not Unique`. Defining Uniqueness is needed in order to create and merge Profiles from various sources without creating duplicate Profiles.
+
```
Comment (evantahler) [inline documents/product/concepts.md:17]: Profile Properties are also given a specific type, letting us know if they are a "number", "date", or "string".  This helps Grouparoo know how to display this data, and send it to Destinations.

File: documents/product/concepts.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+Common examples of `Not Unique` Profile Properties are:
+
+- `First Name`
+- `Last Name`
+- `Date of Last Purchase`
+
+## Profile Property Rules
+
+Profile Property Rules define how a property gets pulled from a Source. Every Source might be structured in slightly different ways, so Grouparoo plugins make it easy for you to define Profile Property Rules.
```
Comment (evantahler) [inline documents/product/concepts.md:33]: For example, you might define a Profile Property Rule called `first_name` which you are pulling from your product database via the `@grouparoo/postgres` plugin, or you might define `number_of_purchases`, which you are using the [Grouparoo Event](https://github.com/grouparoo/grouparoo/blob/master/documents/development/events.md) system to count up.

File: documents/product/concepts.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+## Groups
+
+Groups are a collection of Profiles that you can define and filter. These filters and rules are built to be easy-to-understand by non-technical users.
+
+Let's say we wanted to make a Group of high-value customers who recently abandoned the checkout process. Here is an example of the rules to define this group:
+
+- `[Total Customer Value] [is greater than] [$50]`
+- `[Abandoned Cart] [in the last] [3 days]`
+
+With these rules, you have a Group consisting of every user who has purchased more than \$50 and recently abandoned their cart. You can then synchronize that group in real-time to all of your Destinations to then communicate with them.
```
Comment (evantahler) [inline documents/product/concepts.md:56]: There are 2 types of groups:  `calculated` and `manual`
* Calculated Groups are always kept up to date for you by Grouparoo.  As new Profiles are created, or the Profile Properties of existing Profiles are changed, Group membership will automatically be adjusted.
* Manual Groups do not change.  They contain only the Profiles which you've added.

File: documents/product/concepts.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+## Destinations
+
+Destinations are tools where you'd want Grouparoo Profiles and Groups to be sent. Often these tools are customer marketing tools such as email marketing or push notification tools, though databases and data warehouses can also function at Destinations.
```
Comment (evantahler) [inline documents/product/concepts.md]: so many `tools`!

Perhaps:
> Destinations are where you'd want Grouparoo Profiles and Groups to be sent. Often these destinations are customer marketing tools such as those for email marketing or push notification, though other databases and data warehouses can also function at Destinations.

File: documents/product/concepts.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+- Mailchimp
+- Hubspot
+- Sailthru
+- MySQL
+- PostgreSQL
+- CSV
+
+We are always adding support for more Destinations. If you have a request, please let us know [here](https://www.grouparoo.com/meet)!
+
```
Comment (evantahler) [inline documents/product/concepts.md:82]: Grouparoo will keep the Profile and Group data in your Destinations up-to-date in real time.

File: documents/product/concepts.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+## Sources
+
+Sources are the tools that Grouparoo will connect with to pull in customer data. Sources can be any tool that contains customer data such as data warehouses, databases, and CRM tools.
```
Comment (evantahler) [inline documents/product/concepts.md]: > Sources are the places that Grouparoo will connect with to pull in Profile data. Sources can be any tool that contains customer data such as data warehouses, databases, and other CRM tools.

**Review by evantahler — CHANGES_REQUESTED** [review]
Nice!  I added some comments and suggestions.

Also, please add a link to this page (and I guess a new "Product" section) to the README: https://github.com/grouparoo/grouparoo/blob/master/core/README.md

File: core/README.md [hunk 8]
```diff
@@ -36,6 +36,12 @@ If you are new to Grouparoo, or you want to deploy Grouparoo to your servers, st
 - [📞 Telemetry](https://github.com/grouparoo/grouparoo/blob/master/documents/deployment/telemetry.md)
 - [🩺 Support](https://github.com/grouparoo/grouparoo/blob/master/.github/SUPPORT.md)
 
+### Product Information
+
+You can find more details about the Grouparoo product here.
+
+- [🧠 Grouparoo Product Concepts](https://github.com/grouparoo/grouparoo/blob/master/core/documents/product/concepts.md)
```
Comment (evantahler) [inline core/README.md]: I think the link should be:
```
https://github.com/grouparoo/grouparoo/blob/master/documents/product/concepts.md
``` 

(there's a "core" in there)

File: documents/product/concepts.md [hunk 9]
```diff
@@ -51,12 +55,20 @@ Let's say we wanted to make a Group of high-value customers who recently abandon
 
 With these rules, you have a Group consisting of every user who has purchased more than \$50 and recently abandoned their cart. You can then synchronize that group in real-time to all of your Destinations to then communicate with them.
 
+There are 2 types of groups: **Calculated** and **Manual**
+
+**Calculated Groups** are always kept up to date for you by Grouparoo. As new Profiles are created, or the Profile Properties of existing Profiles are changed, Group membership will automatically be adjusted.
+
+Manual Groups only change when you add or remove Profiles.
```
Comment (evantahler) [inline documents/product/concepts.md]: One of these is bold and the other is not

**Review by andyjih — COMMENTED** [review]
Done!

**Review by evantahler — CHANGES_REQUESTED** [review]

**Review by evantahler — APPROVED** [review]

**Pull request merged**